clang/test/CIR/CodeGen/lambda.cpp

```diff
@@ -18,11 +21,9 @@ void fn() {
 // CHECK-NEXT:     %0 = cir.alloca !ty_anon2E0_, !cir.ptr<!ty_anon2E0_>, ["a"]
 //      CHECK:   cir.call @_ZZ2fnvENK3$_0clEv
 
-// LLVM: {{.*}}void @"_ZZ2fnvENK3$_0clEv"(ptr [[THIS:%.*]])
-// FIXME: argument attributes should be emmitted, and lambda's alignment
-// COM: LLVM: {{.*}} @"_ZZ2fnvENK3$_0clEv"(ptr noundef nonnull align 1 dereferenceable(1) [[THIS:%.*]]){{%.*}} align 2 {
+// LLVM-LABEL:  _ZZ2fnvENK3$_0clEv
 // LLVM: [[THIS_ADDR:%.*]] = alloca ptr, i64 1, align 8
-// LLVM: store ptr [[THIS]], ptr [[THIS_ADDR]], align 8
+// LLVM: store ptr {{%.*}}, ptr [[THIS_ADDR]], align 8
```

Comment:
I see now, you were not using LABEL because you were matching the argument, my bad!

Reply:
You can keep `THIS` for the case were you are matching the arg and use LABEL for the other ones (like I see you did below). Sorry for the churn

Reply:
I don't have the full context here, but if the problem is that `CHECK-LABEL` can't contain variable references, a common pattern I've seen for that, assuming the variable references occur after a label bit, is to use `CHECK-LABEL` for the label bit and then `CHECK-SAME` for the variable references. May or may not be worth it depending on the situation though.

Reply:
Also works

Reply:
Sure, I'll use the `SAME` directive then, Another reason I want to check the argument is (in future) we should check attributes to argument(that's what the COM: line is about), This test seems to be a good case for that.